code/modules/mob/living/carbon/xenomorph/abilities/burrower/burrower_powers.dm

```diff
@@ -98,6 +98,11 @@
 		to_chat(src, SPAN_NOTICE("We must be burrowed to do this."))
 		return
 
+	var/area/current_area = get_area(T)
+	if(current_area.flags_area & AREA_NOTUNNEL)
+		to_chat(src, SPAN_XENOWARNING("There's no way to surface here!!"))
+		return
+
```

Comment:
This is unnecessary; its already done further down at line 129/134 so should revert this change.

The difference is you spend some time *trying* to tunnel there and find out you can't - which is fine.

Reply:
Without that code, it wouldn't work. 
It lets them surface.

Reply:
Did you test this? Because I can comment it out and it still is checked because its a duplicate check as I said.

So either you need to more clearly specify what conditions require this duplicate check; but I don't currently see any situation this isn't simply redundant.

![image](https://github.com/cmss13-devs/cmss13/assets/76988376/5f68862b-59d1-4951-9bb3-4e0f604fcfac)